website/src/lib/auth.js

```diff
@@ -0,0 +1,34 @@
+import { trimTrailingSlash } from './utils';
+const WP_URL = trimTrailingSlash(process.env.WORDPRESS_URL);
+const API_CLIENT_SECRET = process.env.WORDPRESS_HEADLESS_SECRET;
```

Comment:
should this be `process.env.WP_HEADLESS_SECRET` to keep it consistent with the php usage on the backend to avoid confusion? or was this an intentional decision to have it named differently?

Reply:
The convention on the next side is `wordpress_` - since those were set first, should we switch the php ones to `wordpress_`? (I'm not a big fan of over-abbreviating)

Reply:
thats good with me, as long as they all match up!

Reply:
I looked further into this and there isn't an obvious way to go... `WP_ENV` is convention in wordpress PHP, and we've always used `WORDPRESS_API_URL` on the next side. I understand it's inconsistent but am not sure which way to change these to not break compatibility with existing projects